### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by krishna95**

### CODE REVIEW

**Comment by krishna95** [comment]
@hargup

**Comment by hargup** [comment]
Create a new section for research papers.

**Comment by krishna95** [comment]
@hargup

File: README.md [hunk 1]
```diff
@@ -17,9 +17,19 @@ University Courses and MOOCS
...
+
 Books
 -----
 
 * [Feature Extraction and Image Processing](http://users.ecs.soton.ac.uk/msn/book/) by Mark S. Nixon and Alberto S. Aguado
 * [Computer Vision: Algorithms and Applications](http://szeliski.org/Book/) by
   Richard Szeliski
+
+Research Papers
+
```
Comment (hargup) [inline README.md]: Remove this new line to make "Research Papers" a heading, you can see the rendered text from a button on the top of this diff.

**Comment by hargup** [comment]
And a better commit message please.

**Comment by hargup** [comment]
Thanks for the changes.

**Comment by krishna95** [comment]
I should rather thank you to teach me to do each task perfectly and not
take anything lightly.

**Comment by hargup** [comment]
> I should rather thank you to teach me to do each task perfectly and not
> take anything lightly.

Well I did, I think even your new commit message wasn't good enough. And we definitely don't need three commits for this small change, but I didn't ask you squash the commits into one. I'll be more stringent from next time :wink: .

**Comment by krishna95** [comment]
Sure :)

**Pull request merged**